app/routes/events/components/EventDetail/index.js

```diff
@@ -35,7 +36,20 @@ type InterestedButtonProps = {
 
 const InterestedButton = ({ isInterested }: InterestedButtonProps) => {
   const icon = isInterested ? 'star' : 'star-outline';
-  return <Icon className={styles.star} name={icon} />;
+  return (
+    <div style={{ display: 'inline-block' }}>
+      <Tooltip
+        style={{ lineHeight: '1.3rem' }}
+        content={
+          <span style={{ fontSize: '1rem', fontWeight: '100', padding: '0' }}>
+            Favoritt
```

Comment:
Maybe write something about what happens when you follow an event? (It shows up in your calendar)